scripts/cdn_publish.sh

```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
 cdnGravityDir="gravity-ui-web"
 cdnDir="cdn-dist"
-version="$(git describe --abbrev=0 --tags --match *v[0-9]*.[0-9]*.[0-9]*)"
+allVersions=`git tag --list --sort=-committerdate | grep -E '^(gravity-ui-web-)?v[0-9]*.[0-9]*.[0-9]*'`
```

Comment:
@james-nash Should there be something in that cdn code to bail out if it’s _not_ `gravity-ui-web` being committed?

If i updated a version of something else, would that code still run every time? Publishing a non-updated `gravity-ui-web` cdn folder?

Reply:
Good point!

Any chance you could have a creack at it? I'm busy with the new color system PR at the moment.

Reply:
Had a look and a think and as there's extra considerations we can leave this as identified technical debt